azure-pipelines.yml

```diff
@@ -0,0 +1,25 @@
+# Node.js with React
+# Build a Node.js project that uses React.
+# Add steps that analyze code, save build artifacts, deploy, and more:
+# https://docs.microsoft.com/azure/devops/pipelines/languages/javascript
+
+trigger:
+- main
+
+pool:
+  vmImage: ubuntu-latest
+
+steps:
+- task: NodeTool@0
+  inputs:
+    versionSpec: '10.x'
+  displayName: 'Install Node.js'
+
+- script: |
+    ./startserver.sh
+
+- script: |
+    cd score-tracker-frontend
+    npm install
```

Comment:
Let's use yarn install here instead. We want to avoid using multiple package managers if we can.